config/development.json

```diff
@@ -148,6 +148,7 @@
 		"sign-in-with-apple/redirect": true,
 		"signup/add-ons": false,
 		"signup/anchor-fm": true,
+		"signup/design-picker-build-your-own": true,
```

Comment:
Could you rename it to `signup/design-picker-patterns-assembler`? It might be easier to understand what's this